src/util/Permissions.js

```diff
@@ -7,14 +7,14 @@ const { RangeError } = require('../errors');
  */
 class Permissions {
   /**
-   * @param {number|PermissionResolvable[]} permissions Permissions or bitfield to read from
+   * @param {PermissionResolvable|PermissionResolvable[]} permissions Permission(s) to read from
    */
   constructor(permissions) {
     /**
      * Bitfield of the packed permissions
      * @type {number}
      */
-    this.bitfield = typeof permissions === 'number' ? permissions : this.constructor.resolve(permissions);
+    this.bitfield = this.constructor.resolve(permissions);
```

Comment:
revert this... sigh

Reply:
why?

Reply:
Because your change is wrong... It only needs to resolved if the passed in parameter is not a number.

Reply:
Leaving it there allows users to make instances with negative bitfields.

Reply:
Actually you are right. And resolve does handle/throw an error for that...